src/citrine/resources/generative_design_execution.py

```diff
@@ -0,0 +1,114 @@
+"""Resources that represent both individual and collections of design workflow executions."""
+import sys
+from typing import Optional, Union, Iterator
+from uuid import UUID
+
+from citrine._rest.collection import Collection
+from citrine._utils.functions import shadow_classes_in_module
+from citrine._session import Session
+import citrine.informatics.executions.generative_design_execution
+from citrine.informatics.executions.generative_design_execution import GenerativeDesignExecution
+from citrine.informatics.design_candidate import GenerativeDesignExecutionInput
+from citrine.resources.response import Response
+
+
+shadow_classes_in_module(
+    citrine.informatics.executions.generative_design_execution, sys.modules[__name__]
+)
+
+
+# TODO: Double check the path.
+class GenerativeDesignExecutionCollection(Collection["GenerativeDesignExecution"]):
+    """A collection of GenerativeDesignExecutions."""
+
+    _path_template = '/projects/{project_id}/generative-design/executions'
+    _individual_key = None
+    _collection_key = 'response'
+    _resource = GenerativeDesignExecution
+
+    def __init__(self, project_id: UUID, session: Session):
+        self.project_id: UUID = project_id
+        self.session: Session = session
+
+    def build(self, data: dict) -> GenerativeDesignExecution:
+        """Build an individual GenerativeDesignExecution."""
+        execution = GenerativeDesignExecution.build(data)
+        execution._session = self.session
+        execution.project_id = self.project_id
+        return execution
+
+    def trigger(self, generative_design_execution_input: GenerativeDesignExecutionInput):
+        """Trigger a Generative Design execution given a score."""
+        path = self._get_path()
+        request_dict = generative_design_execution_input.dump()
+        data = self.session.post_resource(path, request_dict)
+        return self.build(data)
+
+    def register(self, model: GenerativeDesignExecution) -> GenerativeDesignExecution:
+        """Cannot register an execution."""
+        raise NotImplementedError("Cannot register a GenerativeDesignExecution.")
+
+    def update(self, model: GenerativeDesignExecution) -> GenerativeDesignExecution:
+        """Cannot update an execution."""
+        raise NotImplementedError("Cannot update a GenerativeDesignExecution.")
+
+    def archive(self, uid: Union[UUID, str]):
```

Comment:
`archive` and `restore` are not defined on `Collection`, so there's no need to overwrite them.